src/machine/usb/midi/midi.go

```diff
@@ -32,12 +32,12 @@ func newMidi() *midi {
 	m := &midi{
 		buf: NewRingBuffer(),
 	}
-	machine.EnableMIDI(m.Callback, m.CallbackRx, nil)
+	machine.EnableMIDI(m.Handler, m.rxHandler, nil)
```

Comment:
RxHandler is correct here.

```suggestion
	machine.EnableMIDI(m.Handler, m.RxHandler, nil)
```

Reply:
Since there is already a `RxHandler()` method, the internal field cannot be exported. Otherwise you end up with error:

```
/home/ron/Development/tinygo/tinygo/src/machine/usb/midi/midi.go:75:16: field and method with the same name RxHandler
```

Reply:
In other words, I think it is correct the way I have it.

Reply:
@deadprogram 
Since `m.rxHandler` at the time `machine.EnableMIDI` is called is nil, the callback is no longer called.
`examples/usb-midi` toggles LEDs whenever MIDI is received.

https://mikatahara.github.io/SendReceive/

![image](https://user-images.githubusercontent.com/9251039/178853867-2c056f01-00af-4274-aabc-0bddef1c83a1.png)

Reply:
You are certainly correct @sago35 I got confused by the naming :smiley_cat: 

Corrected in PR https://github.com/tinygo-org/tinygo/pull/2988